{"tag":"AnalysisSuccess","contents":{"requestedTools":null,"toolResults":[[{"tag":"Infer"},{"tag":"ToolFailure","contents":{"tag":"ToolSurfaceableFailure","contents":["BuildError while running Infer on master: Is the build broken?","[DEBUG] [28/May/2019:20:08:10 +0000] [()] Analysis is starting.\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Getting analysis results for commit \"master\"\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Reading config file\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Final, parsed, repository configuration structure:\n    ProjSet\n      { buildCommand = Nothing\n      , argsList = []\n      , interestingIssue = <function>\n      , setupScript = Nothing\n      , toolList = Nothing\n      , customTools = []\n      , whitelist = Nothing\n      , jdk11 = False\n      }\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Determining tools...\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]  infer ENABLED\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Trying to autodetect the build system based on files present in the repository.\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]  errorprone ENABLED\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]  hlint DISABLED, not applicable\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] 2 applicable tools\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Sanitizing repository\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Running infer on branch master\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]   Running infer bulk with a non-explicit build system.\n[DEBUG] [28/May/2019:20:08:52 +0000] [()] Infer BuildError: Src branch: All build attempts failed: Capture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n20:08:52.674 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred configuring project ':examples:DaoExample'.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 2s\\nERROR: couldn't run compilation command `[u'./gradlew', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-7362b798ce234c25 --out\\n  /tmp/analyzing-7362b798ce234c25/infer-out -- ./gradlew assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\nCapture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n20:08:17.924 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.924 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Where:\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file '/tmp/analyzing-7362b798ce234c25/build.gradle' line: 4\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred evaluating root project 'analyzing-7362b798ce234c25'.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Cannot add task 'wrapper' as a task with that name already exists.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 4s\\nERROR: couldn't run compilation command `[u'gradle', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-7362b798ce234c25 --out\\n  /tmp/analyzing-7362b798ce234c25/infer-out -- gradle assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\n\n"]}}],[{"tag":"ErrorProne"},{"tag":"ToolFailure","contents":{"tag":"ToolSurfaceableFailure","contents":["Execution Failed: ---STDOUT---\n---ERR---\n---CMD---\nCreateProcess {cmdspec = RawCommand \"gradle\" [\"-q\",\"-I\",\"/tmp/analyzing-7362b798ce234c25/muse-compdb.gradle\",\"assemble\"], cwd = Just \"/tmp/analyzing-7362b798ce234c25\", env = Nothing, std_in = Inherit, std_out = Inherit, std_err = Inherit, close_fds = False, create_group = False, delegate_ctlc = False, detach_console = False, create_new_console = False, new_session = False, child_group = Nothing, child_user = Nothing, use_process_jobs = False}","[DEBUG] [28/May/2019:20:08:10 +0000] [()] Analysis is starting.\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Getting analysis results for commit \"master\"\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Reading config file\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Final, parsed, repository configuration structure:\n    ProjSet\n      { buildCommand = Nothing\n      , argsList = []\n      , interestingIssue = <function>\n      , setupScript = Nothing\n      , toolList = Nothing\n      , customTools = []\n      , whitelist = Nothing\n      , jdk11 = False\n      }\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Determining tools...\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]  infer ENABLED\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Trying to autodetect the build system based on files present in the repository.\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]  errorprone ENABLED\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]  hlint DISABLED, not applicable\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] 2 applicable tools\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Sanitizing repository\n[DEBUG] [28/May/2019:20:08:10 +0000] [()] Running infer on branch master\n[DEBUG] [28/May/2019:20:08:10 +0000] [()]   Running infer bulk with a non-explicit build system.\n[DEBUG] [28/May/2019:20:08:52 +0000] [()] Infer BuildError: Src branch: All build attempts failed: Capture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n20:08:52.674 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred configuring project ':examples:DaoExample'.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n20:08:52.675 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 2s\\nERROR: couldn't run compilation command `[u'./gradlew', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-7362b798ce234c25 --out\\n  /tmp/analyzing-7362b798ce234c25/infer-out -- ./gradlew assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\nCapture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n20:08:17.924 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.924 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Where:\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file '/tmp/analyzing-7362b798ce234c25/build.gradle' line: 4\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred evaluating root project 'analyzing-7362b798ce234c25'.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Cannot add task 'wrapper' as a task with that name already exists.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n20:08:17.925 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 4s\\nERROR: couldn't run compilation command `[u'gradle', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-7362b798ce234c25 --out\\n  /tmp/analyzing-7362b798ce234c25/infer-out -- gradle assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\n\n[DEBUG] [28/May/2019:20:08:52 +0000] [()] Sanitizing repository\n[DEBUG] [28/May/2019:20:08:52 +0000] [()] Trying to autodetect the build system based on files present in the repository.\n[DEBUG] [28/May/2019:20:08:52 +0000] [()] Using build systems of [Gradlew [\"assemble\"],BuildGradle [\"assemble\"]] to build compilation database.\n[DEBUG] [28/May/2019:20:08:52 +0000] [()] Wrote the plugin for use with ./gradlew -I /tmp/analyzing-7362b798ce234c25/muse-compdb.gradle assemble\n[DEBUG] [28/May/2019:20:08:55 +0000] [()] Wrote the plugin for use with gradle -I /tmp/analyzing-7362b798ce234c25/muse-compdb.gradle assemble\n[DEBUG] [28/May/2019:20:08:57 +0000] [()] <function> BuildError: BuildError {renderError = \"Execution Failed: ---STDOUT---\\n---ERR---\\n---CMD---\\nCreateProcess {cmdspec = RawCommand \\\"gradle\\\" [\\\"-q\\\",\\\"-I\\\",\\\"/tmp/analyzing-7362b798ce234c25/muse-compdb.gradle\\\",\\\"assemble\\\"], cwd = Just \\\"/tmp/analyzing-7362b798ce234c25\\\", env = Nothing, std_in = Inherit, std_out = Inherit, std_err = Inherit, close_fds = False, create_group = False, delegate_ctlc = False, detach_console = False, create_new_console = False, new_session = False, child_group = Nothing, child_user = Nothing, use_process_jobs = False}\"}\n"]}}]]}}
